### ISSUE TITLE
A11Y | The "Notification" icon button does not have a valid accessible label.

### ISSUE THREAD

**Issue by hilcurtis** [Esri/global-nav#331]
**ID**
1843920471

**Violation**
Provide a valid label for form fields

**Module Name**
03a Header Menu - Top Row

**Description**
[Issue]
The "Notification" icon button does not have a valid accessible label. 

[User Impact]
When buttons do not explicitly include a label, assistive technologies may incorrectly render the label or provide no label at all to users. When labels are not present or are incorrect, users of assistive technologies may not be able to complete an action.

[Code Reference]
`<button aria-haspopup="true" aria-expanded="false" aria-controls="esri-header-notifications-menu" id="esri-header-notifications-control" class="esri-header-notifications-control"><span class=""><svg id="esri-header-notifications-image" style="transform: rotate(360deg);" class="esri-header-notifications-image"><path d="M18 16v-5.087A5.91 5.91 0 0 0 13.59 5.2a2 2 0 1 0-3.18 0A5.91 5.91 0 0 0 6 10.913V16a3 3 0 0 1-3 3v1h18v-1a3 3 0 0 1-3-3zM12 3a1 1 0 1 1-1 1 1.001 1.001 0 0 1 1-1zM5.643 19A3.992 3.992 0 0 0 7 16v-5.087A4.919 4.919 0 0 1 11.913 6h.174A4.919 4.919 0 0 1 17 10.913V16a3.992 3.992 0 0 0 1.357 3zM13 21h1a2 2 0 0 1-4 0h1a1 1 0 0 0 2 0z"></path></svg></span></button>`

**Note**
[Recommendation]
Developer should provide an accessible label to the actionable icon buttons. Developer can use an aria-label to provide a label to the "notification" icon. 

[Compliant Code Example]
`<button aria-haspopup="true" aria-expanded="false" aria-label="notification" aria-controls="esri-header-notifications-menu" id="esri-header-notifications-control" class="esri-header-notifications-control">
`
![image](https://user-images.githubusercontent.com/34460455/100175635-a47a3d00-2e83-11eb-85aa-9b7acf4ce4f5.png)
